### PR TITLE
Review ch7

### PR DESCRIPTION
**Pull request by bbdaniels**

### CODE REVIEW

File: chapters/7-publication.tex [hunk 1]
```diff
@@ -432,6 +441,13 @@ \section{Preparing a complete reproducibility package}
 Major journals now often require that you provide both the data and code required to recreate your results.
 Some even require being able to reproduce the results themselves
 before they will approve a paper for publication.\sidenote{\citet{vilhuber2020report}}
+If you are producing a policy output,
+such as an open policy analysis\sidenote{\citet{hoces2018framework}}
+or some other type of material,\sidenote{Such as \url{https://github.com/worldbank/sdgatlas2018} or
+  \url{https://blogs.worldbank.org/opendata/how-we-mass-produced-reproducible-human-capital-project-country-briefs}}
```
Comment (luizaandrade) [inline chapters/7-publication.tex:447]: This is also a good example of a dissemination piece with a reproducibility package: https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility

**Comment by luizaandrade** [comment]
@bbdaniels , I made some minor suggestions in 74998e8, the most relevant of them on the README. Please take a look and see if you agree. Otherwise, I think it's good to merge. This is a great chapter!

File: chapters/7-publication.tex [hunk 2]
```diff
@@ -602,6 +634,16 @@ \subsection{Releasing a reproducibility package}
...
 and organizations limit access to platforms other than the one officially adopted.
 
+% README
+Finally, any reproducibility package should include an overview of its contents
+and instructions on how to recreate your outputs.
+This is typically done in the form of a README file.
+A good README will guide the reader through all the items included in the package.
+Fortunately, a very good template for such documents is offered by
+a consortium of social science data editors.\sidenote{
+	\url{https://social-science-data-editors.github.io/template_README/}
```
Comment (bbdaniels) [inline chapters/7-publication.tex:644]:
```suggestion
	\url{https://doi.org/10.5281/zenodo.4319999}
```

File: chapters/7-publication.tex [hunk 3]
```diff
@@ -432,6 +444,13 @@ \section{Preparing a complete reproducibility package}
 Major journals now often require that you provide both the data and code required to recreate your results.
 Some even require being able to reproduce the results themselves
 before they will approve a paper for publication.\sidenote{\citet{vilhuber2020report}}
+If you are producing a policy output,
+such as an open policy analysis\sidenote{\citet{hoces2018framework}}
+or some other type of material,\sidenote{Such as \url{https://github.com/worldbank/sdgatlas2018} or
+  \url{https://blogs.worldbank.org/opendata/how-we-mass-produced-reproducible-human-capital-project-country-briefs}}
```
Comment (bbdaniels) [inline chapters/7-publication.tex:450]:
```suggestion
If you are producing a policy output,
such as an open policy analysis\sidenote{\citet{hoces2018framework}}
or some other type of material,\sidenote{Such as \url{https://github.com/worldbank/sdgatlas2018}, \url{https://blogs.worldbank.org/opendata/how-we-mass-produced-reproducible-human-capital-project-country-briefs}, or \url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}
```

**Pull request merged**